cql3/values.hh

```diff
@@ -54,6 +56,7 @@ struct raw_value_view {
     raw_value_view(fragmented_temporary_buffer::view data)
         : _data{data}
     {}
+    raw_value_view(bytes&& temporary_storage);
```

Comment:
explicit.

Reply:
these constructors are private, that's probably why nobody bothered to mark them explicit. Doesn't hurt though, I'll add explicit in v2.